requirements.txt

```diff
@@ -60,6 +60,7 @@ disposable-email-domains>=0.0.52
 gevent==1.4.0
 supervisor==4.1.0
 supervisor_checks==0.8.1
+urllib3==1.25.7
```

Comment:
Are you sure we need this as an explicit dependency?

Reply:
Hmm - I think it addresses a bug raised with in a couple of places. I'll try it with just specification of the `pyopenssl` version.

Reply:
OK tested it and specify of pyopenssl is enough so I have requirement requirement for specified urllib.

Reply:
If it's because of #4526 then #4567 should be enough. But I'll check release notes for the other libs to see if there is anything risky there.

Reply:
The `cryptography` upgrade looks safe, but `pyopenssl` has some deprecations which I'm not sure if they are supported by all the other libs we depend on.

Reply:
I'll try rerunning with just `cryptography` tomorrow morning - Berlin time.

Reply:
Thanks! Appreciate the effort.

Reply:
It worked with `cryptography` 2.8 and `pyopenssl` 19.0.0 so I've made that commit.

Reply:
Great. I fixed a merge conflict and once the build finishes will merge :+1: